code/modules/client/client defines.dm

```diff
@@ -60,6 +60,7 @@
 	var/karma = 0
 	var/karma_spent = 0
 	var/karma_tab = 0
+	var/wants_karma = TRUE // let's just keep it true as default for obvious reasons
```

Comment:
Use 1 and 0

Reply:
Why 1 and 0? this is a boolean var, shouldn't TRUE and FALSE be used for those

Reply:
Yeah that's what I'm saying, wtf? Isn't `TRUE` & `FALSE` the standard that's being applied most everywhere else?

Reply:
Ignore my retardation